feat: megacommit

VincentRPS opened the pull request.

steamweaver commented:
Hello! I am your friendly Train Conductor here in station Derailed to inform you that you must sign our [Contributor License Agreement](https://gist.github.com/VincentRPS/3b0f267edfa23e365b03b8bbc2f0cc6e) before preceding. Once you've finished reading, and agree to everything there, comment 'I have read the CLA Document and I hereby sign the CLA' so we can merge your PR!

Go Fix Some Code, Train Conductor
   - - -
   I have read the CLA Document and I hereby sign the CLA
   - - -
   <sub>You can retrigger this bot by commenting **recheck** in this Pull Request</sub>

THEGOLDENPRO commented:
I have read the CLA Document and I hereby sign the CLA

THEGOLDENPRO commented:
I have read the CLA Document and I hereby sign the CLA

steamweaver commented:
Hello! I am your friendly Train Conductor here in station Derailed to inform you that you must sign our [Contributor License Agreement](https://gist.github.com/VincentRPS/3b0f267edfa23e365b03b8bbc2f0cc6e) before preceding. Once you've finished reading, and agree to everything there, comment 'I have read the CLA Document and I hereby sign the CLA' so we can merge your PR!

Go Fix Some Code, Train Conductor
   - - -
   I have read the CLA Document and I hereby sign the CLA
   - - -
   <sub>You can retrigger this bot by commenting **recheck** in this Pull Request</sub>

THEGOLDENPRO commented:
I don't like this train conductor, can the train inspector bring their boss.

VincentRPS commented:
recheck

steamweaver commented:
Hello! I am your friendly Train Conductor here in station Derailed to inform you that you must sign our [Contributor License Agreement](https://gist.github.com/VincentRPS/3b0f267edfa23e365b03b8bbc2f0cc6e) before preceding. Once you've finished reading, and agree to everything there, comment 'I have read the CLA Document and I hereby sign the CLA' so we can merge your PR!

Go Fix Some Code, Train Conductor
   - - -
   I have read the CLA Document and I hereby sign the CLA
   - - -
   <sub>You can retrigger this bot by commenting **recheck** in this Pull Request</sub>

VincentRPS committed:
feat: some new message stuff

VincentRPS committed:
Merge branch 'master' of https://github.com/derailedapp/derailed

VincentRPS committed:
feat: megacommit

Includes a temporary dummy app only for design purposes.

- New API routes for deleting, editing, and mark messages as read.
